src/ensembl/production/core/models/hive.py

```diff
@@ -327,13 +330,43 @@ def get_result_for_job(self, job, progress=False):
             else:
                 result['status'] = self.get_job_tree_status(job)
                 if progress:
-                    result['progress'] = self.get_jobs_progress(job)
+                    result['progress'] = self.get_all_jobs_progress(job.job_id, analysis_id=analysis_id)
+                    #result['progress'] = self.get_jobs_progress(job)
```

Comment:
Could you remove as much as possible commented code? it's often misleading for the one who will come after you.

Reply:
sure i will remove the comment code